pluginProjects/FileLoaders/VESPERS2011XRFFileLoader/VESPERS2011XRFFileLoaderPlugin.cpp

```diff
@@ -69,6 +71,10 @@ bool VESPERS2011XRFFileLoaderPlugin::load(AMScan *scan, const QString &userDataF
 			// Note!  Not general!
 			QList<AMAxisInfo> axisInfo;
 			AMAxisInfo ai("Energy", 2048, "Energy", "eV");
+
+			if (elements == 32)
+				ai.size = 4096;
+
```

Comment:
Yeah.  What I should really do is make a new one since these XRF scans are actually incredibly general.